raw_scripts/STOC_bird_temporal_tendency/2019/makeTableAnalyseGalaxy.r

```diff
@@ -10,17 +10,17 @@
 args = commandArgs(trailingOnly=TRUE)
 
 if (length(args)==0) {
-    stop("At least one argument must be supplied (input file)", call.=FALSE) #if no args -> error and exit1
+    stop("At least one argument must be supplied (input file)", call.=FALSE) #si pas d'arguments -> affiche erreur et quitte / if no args -> error and exit1
 } else {
-    ImportduSTOC<-args[1] #Ne pas utiliser de "" pour la déclaration de variable, je n'avais jamais vu ça. c'est un vrai truc de R ? non une erreur de debutant ! ###### Nom du fichier sans extension ".typedefichier"  / file name without the file type ".filetype"    
+    ImportduSTOC<-args[1] ###### Nom du fichier importé depuis la base de données STOCeps sans extension ".typedefichier"  / file name imported from the STOCeps database without the file type ".filetype"    

```

Comment:
Can we add English comments :)

Reply:
Can't hide anything from you :D 
Yes of course we will, for now it's more like a github course with @BenYguel

Reply:
Sorry, I will shut up now :)

Reply:
Please don't !